src/GeoCoordinate/GeoCoordinate.cs

```diff
@@ -0,0 +1,311 @@
+﻿using System;
+using System.Globalization;
+#if NETFX_CORE
```

Comment:
I think the more modern term would be #if WINDOWS_UWP

Reply:
is `WINDOWS_UWP` true for WinUI 3?

Reply:
Is NETFX_CORE true for WinUI3 ?  Which one should we use?  If it's NETFX_CORE then it's ok :)

Reply:
Just tested and seems like they're both true in WinUI 3 pr2 😄 So, never mind...